### ISSUE TITLE
No module named 'casingSimulations'

### ISSUE THREAD

**Issue by YingHuuu** [simpeg-research/lbnl-2019-resistive-casing#2]
I'm trying to run these notebooks, but I found this error information in jupyter notebook:

`import casingSimulations as casing_sim`

I noticed the failed signal from Travis check about last commit. The error information in Travis job log is:

```
Pip subprocess output:
Collecting git+https://github.com/simpeg/simpeg.git@simulation (from -r /home/travis/build/simpeg-research/lbnl-2019-resistive-casing/condaenv.x_p2do2l.requirements.txt (line 1))
  Cloning https://github.com/simpeg/simpeg.git (to revision simulation) to /tmp/pip-req-build-yqcqcgss
Pip subprocess error:
  Running command git clone -q https://github.com/simpeg/simpeg.git /tmp/pip-req-build-yqcqcgss
  WARNING: Did not find branch or tag 'simulation', assuming revision or ref.
  Running command git checkout -q simulation
  error: pathspec 'simulation' did not match any file(s) known to git
ERROR: Command errored out with exit status 1: git checkout -q simulation Check the logs for full command output.
CondaEnvException: Pip failed
```

The error is about `pip`. Maybe there is no branch tagged 'simulation' or 'casingSimulations' in the repositories. I post related codes about `pip` in `environment.yml` configuration file here.

```
  - pip:
    - git+https://github.com/simpeg/simpeg.git@simulation
    - git+https://github.com/simpeg-research/casingSimulations.git@simulation
```

**Comment by YingHuuu** [comment]
I found a way to solve this problem: 

use `pip install git+https://github.com/simpeg-research/casingSimulations.git@simulation` in conda prompt

I'm a green hand, I don't know if I understand this problem in a right way...😅

**Comment by YingHuuu** [comment]
I also checked the branches both in SimPEG and simpeg-research repositories just now. There is no branch tagged 'simulation' in SimPEG repository. And there is a branch tagged 'simulation' in simpeg-research/casingSimulations. So the error is about `git+https://github.com/simpeg/simpeg.git@simulation`.

I followed intruction in SimPEG docs and install SimPEG from conda-forge to solve the error caused by `git+https://github.com/simpeg/simpeg.git@simulation`.

**Comment by lheagy** [comment]
Thanks for catching this @winnerer123! Yes, we merged the `simulation` branch of SimPEG and released [SimPEG 0.14.0](https://github.com/simpeg/simpeg/releases/tag/v0.14.0).  I have updated casingSimulations (simpeg-research/casingSimulations#14). 

This should close with #3
